lib/rest-client.js

```diff
@@ -22,17 +22,52 @@ module.exports = class RestClient {
     return await this.request.post('/open-api/order/create', params);
   }
 
+  async placeActiveOrderV2(params) {
+    assert(params, 'No params passed');
+    assert(params.side, 'Parameter side is required');
+    assert(params.symbol, 'Parameter symbol is required');
+    assert(params.order_type, 'Parameter order_type is required');
+    assert(params.qty, 'Parameter qty is required');
+    assert(params.time_in_force, 'Parameter time_in_force is required');
+
+    if(params.order_type === 'Limit') assert(params.price, 'Parameter price is required for limit orders');
+
+    return await this.request.post('/v2/private/order/create', params);
+  }
+
   async getActiveOrder(params) {
     return await this.request.get('/open-api/order/list', params);
   }
 
   async cancelActiveOrder(params) {
     assert(params, 'No params passed');
-    assert(params.order_id, 'Parameter order_id is required');
+    assert(params.order_id || params.order_link_id, 'Parameter order_id OR order_link_id is required');
 
     return await this.request.post('/open-api/order/cancel', params);
   }
 
+  async cancelActiveOrderV2(params) {
```

Comment:
I'd suggest to replace `cancelActiveOrder` with this one, as the docs recommend to use v2.